project/Newevaluation.html

```diff
@@ -110,253 +143,221 @@
-      width: 15%;
-      text-align: left;
-      font-family: roboto;
+      width: 10%;
     }
 
 
-    //.firstLi {
-    //  padding-left: 10px;
-      //margin-left: 0px;
-    //}
 
-    //.firstSel {
-    //  padding-left: 10px;
-    //}
+/* ------------------------------------ HTTP ---------------------------------*/
 
     .sixthBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 200px;
-      padding-top: 30px;
+      margin-top: -2rem;
+      width: 100%;
+      height: 16rem;
+      padding-top: 2rem;
     }
 
+
+/* --------------------------------- HTML ------------------------------------*/
+
     .seventhBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 200px;
-      padding-top: 30px;
+      margin-top: -2rem;
+      width: 100%;
+      height: 16rem;
+      padding-top: 2rem;
     }
 
+
+/* ---------------------------------- CSS ------------------------------------*/
+
     .eigthBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 200px;
-      padding-top: 30px;
+      margin-top: -2rem;
+      width: 100%;
+      height: 16rem;
+      padding-top: 2rem;
     }
 
+
+/* ---------------------------------- Javascript------------------------------*/
+
     .ninethBox {
       background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 350px;
-      padding-top: 30px;
+      margin-top: -2rem;
+      width: 100%;
+      height: 24rem;
+      padding-top: 2rem;
     }
 
-    .lastBox {
-      background-color: whitesmoke;
-      margin-top: -21px;
-      width: 1895px;
-      height: 200px;
-      padding-top: 30px;
+    .JS {
+      height: 80%;
     }
 
-    .candidate {
-      width: 380px;
-      height: 40px;
-      border: none;
-      font-size: 18px;
-      color: #bfbfbf;
-      border-bottom: 1px solid lightgrey;
-    }
 
-    .interviewer {
-      width: 380px;
-      height: 40px;
-      border: none;
-      font-size: 18px;
-      color: #bfbfbf;
-      border-bottom: 1px solid lightgrey;
-    }
 
-    .date {
-      width: 380px;
-      height: 40px;
-      border: none;
-      font-size: 18px;
-      color: rgb(82, 86, 89);
-      border-bottom: 1px solid lightgrey;
-    }
+/*---------------------------------- NodeJS --------------------------------- */
 
-    .JS {
-      height: 80%;
+    .lastBox {
+      background-color: whitesmoke;
+      margin-top: -2rem;
+      width: 100%;
+      height: 14rem;
+      padding-top: 2rem;
     }
 
+
+
+/* --------------------------- Submit Button ---------------------------------*/
+
     input[type=submit] {
-      width: 130px;
+      width: 10rem;
       border: none;
-      height: 60px;
+      height: 4rem;
       background-color: #03A9F4;
       box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.5), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-      margin-left: 1250px;
+      margin-left: 87.9rem;
```

Comment:
`px` are usually easier to work with, indeed. `rem` should work reasonably well, too, but there is still that relativity to the `font-size` on the root element. Just as long as you properly plan things out, either option should do the job just fine. ;)